taxonomies/ingredients.txt

```diff
@@ -5421,7 +5421,6 @@ pt:fermentos lácticos e de maturação
 en:selected lactic ferments
 de:ausgewählte Milchsäurekulturen
 fr:ferments lactiques sélectionnés
-nl:geselecteerde fermenten
```

Comment:
Looks ok to me, otherwise the other entry should be removed.

Reply:
the other instance is under
```<en:ferments
en:selected ferments
de:ausgewählte Kulturen
es:fermentos seleccionados
fr:ferments sélectionnés
it:fermenti selezionati
nl:geselecteerde fermenten
pt:fermentos selecionados
# ingredient/fr:ferments-sélectionnés has 134 products in 7 languages @2019-07-08
```
which seems like the more accurate translation.

Reply:
Yes you are right. You could change it to `geselecteerde melkzuurfermenten`.